inspire_utils/name.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, name, constants=None):
             self._parsed_name = HumanName(name, constants=constants)
             self._parsed_name.capitalize()
 
-        if ',' not in name and ('.' not in self.first_list[-1] or len(self.first_list[-1]) == 1):
+        if ',' not in name and (not self.first_list or (self.first_list and ('.' not in self.first_list[-1] and not len(self.first_list[-1]) == 1))):
```

Comment:
this also seems different what was there before (`and not` instead of `or`). Why?

Reply:
Because we would like to check if it is not initial -> if it's of length 1 it is initial. For example first_list might be `['David', 'E']` and in this case `maybe_only_last_name` should be false (so the condition should return false)

Reply:
no, nothing prevents `E` from being a last name. Let's just leave this condition out then.